arquivo_estaticos_no_django.md

```diff
@@ -0,0 +1,23 @@
+# Arquivos estáticos no Django
+
+Finalmente nós iremos ver mais de perto nessas coisas que chamamos de **arquivos estáticos**. 
+Arquivos estáticos são todas as suas imagens e arquivos CSS -- arquivos que não são dinâmicos, então seu conteúdo não depende do contexto da requisição e será o mesmo para todos os usuários.
+
+## Onde colocar os arquivos estáticos para Django
+
+Django já sabe onde encontrar os arquivos estáticos para o built-in "admin" app. Agora só precisamos adicionar alguns arquivos estáticos para nosso próprio app, ```blog```.
```

Comment:
Como traduzir `built-in admin app`? 🤔

Reply:
Podemos chamar de padrão pois admin é uma `app` padrão do Django.
`... para a app padrão do Django 'admin'`